build.gradle.kts

```diff
@@ -71,6 +71,7 @@ val enableSpotBugs = props.bool("spotbugs")
 val enableCheckerframework by props()
 val enableErrorprone by props()
 val enableDependencyAnalysis by props()
+val enableParameters by props(true)
```

Comment:
The intention was enabled by default features are named `skip*` and disabled by default are named `enable*`.

So I would suggest rephrasing the flag as `skipJavacParameterNames` or something like that.

WDYT?